invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/CommunitiesListModal/CommunityItem.js

```diff
@@ -0,0 +1,57 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2023 CERN.
+//
+// InvenioRDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import _truncate from "lodash/truncate";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import React from "react";
+import PropTypes from "prop-types";
+
+import { Item, Label, Icon, Button } from "semantic-ui-react";
+import { Image } from "react-invenio-forms";
+
+export const CommunityItem = ({ result }) => (
+  <Item key={result.id} verticalAlign="middle" className="justify-space-between">
+    <Image size="mini" src={result.links.logo} />
+
+    <Item.Content verticalAlign="middle">
+      <Item.Header as="h3" className="ui small header flex align-items-center mb-5">
+        <a href={`/communities/${result.slug}`} className="p-0">
+          {result.metadata.title}
+        </a>
+
+        {result.access.visibility === "restricted" && (
```

Comment:
maybe we could make it a common component in communities ? I think we use it many times around so we wouldn't have to copy the code